packages/autotest/src/server/RouteHandler.ts

```diff
@@ -240,6 +226,17 @@ export default class RouteHandler {
             };
             const dockerReq = http.request(reqOptions, handler);
             dockerReq.end(0);
+//             Log.info("RouteHandler::getDockerImage(..) - Preparing to build image");
```

Comment:
@braxtonhall 

Did we want this commented out code in here? I wonder if we could take them out if they are no longer being used.

Reply:
I think that's a questions for @LucasZamprogno 
I didn't work on this bit

Reply:
Ah, roger that. Thanks!

Reply:
Sorry for being slow/not seeing stuff, I don't get notifications for github emails. It's fine to remove, I can always go back to it if I want to take another stab at it.

Reply:
Thanks Lucas! Will do.